ruff.toml

```diff
@@ -3,7 +3,6 @@ target-version = "py311"
 lint.select = ["ALL"]
 
 lint.ignore = [
-    "ANN101",   # Missing type annotation for `self` in method
```

Comment:
`warning: The following rules have been removed and ignoring them has no effect:
    - ANN101`